src/System.IO.Ports/src/System/IO/Ports/SerialStream.cs

```diff
@@ -714,13 +716,7 @@ internal int BytesToWrite
 
                 // prep. for starting event cycle.
                 _eventRunner = new EventLoopRunner(this);
-
-                // We should consider migrating to a Task here rather than simple background Thread
-                // This would let any exceptions be marshalled back to the owner of the SerialPort
-                // Some discussion in GH issue #17666
-                Thread eventLoopThread = new Thread(_eventRunner.SafelyWaitForCommEvent);
-                eventLoopThread.IsBackground = true;
-                eventLoopThread.Start();
+                _waitForComEventTask = Task.Run(() => _eventRunner.SafelyWaitForCommEvent());
```

Comment:
We don't need to use the 'SafelyWait..' wrapper either, because that was just there to protect the process from unhandled exceptions on background threads.  With a task we'll get the exceptions back properly.